modules/core/keeper/msg_server_test.go

```diff
@@ -870,8 +870,7 @@ func (suite *KeeperTestSuite) TestChannelUpgradeInit() {
 	cases := []struct {
 		name      string
 		malleate  func()
-		expResult func(res *channeltypes.MsgChannelUpgradeInitResponse, err error)
-		expEvents bool
+		expResult func(ctx sdk.Context, res *channeltypes.MsgChannelUpgradeInitResponse, err error)
```

Comment:
what do you think about adding the events as the argument instead of passing the full context. 

e.g. `expResult func(res *channeltypes.MsgChannelUpgradeInitResponse, err error, events abci.Events)`

and pass it like `tc.ExpResult(res, err, ctx.EventManager().Events().ToABCIEvents())`

I think this way might be more intuitive

Reply:
Guessing the only upside of `ctx` would be if in the future we'd might want to check for contextual stuff, can't see how that would be tho, dig both options.

Reply:
yea that makes sense also! was thinking along the lines of @DimitrisJim but i suspect you're right for at least the foreseeable future. will update to just pass in events.